### PR TITLE
Add NVIDIA GPU driver link

### PR DESCRIPTION
**Pull request by SohamMalakar**
Adding the `NVIDIA-Linux-[...].run` download link.

### CODE REVIEW

**Comment by maximilianovermeyer** [comment]
You could remove the explicit language parameter in the URL.

File: README.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ as root:
 Note that the kernel modules built here must be used with gsp.bin
 firmware and user-space NVIDIA GPU driver components from a corresponding
 515.48.07 driver release.  This can be achieved by installing
-the NVIDIA GPU driver from the .run file using the `--no-kernel-modules`
+the [NVIDIA GPU driver](https://www.nvidia.com/Download/index.aspx?lang=en-us) from the .run file using the `--no-kernel-modules`
```
Comment (ViniciusFXavier) [inline README.md:21]:
```suggestion
the [NVIDIA GPU driver](https://www.nvidia.com/Download/index.aspx) from the .run file using the `--no-kernel-modules`
```

**Comment by SohamMalakar** [comment]
> You could remove the explicit language parameter in the URL.

sure why not

**Review by ViniciusFXavier — COMMENTED** [review]
👍

**Pull request closed without merging**